scrapers/stackoverflow.js

```diff
@@ -0,0 +1,61 @@
+var unirest = require("unirest");
+var question = "";
+for (i = 2; i < process.argv.length; i++) {
+    question += process.argv[i] + " ";
+}
+var url = "https://api.stackexchange.com/2.2/search/advanced?key=jAAMPr3uTSgYulMJ3LY5Hw((&order=desc&sort=activity&q=" + question + "&answers=1&site=stackoverflow";
```

Comment:
Yes, please remove the API key. Without API key it is 300 per day ([link](https://stackapps.com/questions/3055/is-there-a-limit-of-api-requests)). So, why not make it optional (as loklak server can be built and used personally, in case the users use it personally, they can provide their API key and harvest more data, similar to [KaizenHarvester](https://github.com/loklak/loklak_server/blob/9e3a4ebe8887304e1afae3c58e1cba27fbc87d62/src/org/loklak/harvester/strategy/KaizenHarvester.java#L67-L71) in loklak_server).
And you can add a try/catch if the API way fails, go for the scraping way.
@singhpratyush @vibhcool @djmgit @hemantjadon What do you suggest ?